gensim/models/wrappers/wordrank.py

```diff
@@ -87,28 +88,26 @@ def train(cls, wr_path, corpus_file, out_name, size=100, window=15, symmetric=1,
         `ensemble` = 0 (default), use ensemble of word and context vectors
         """
 
-        meta_data_path = 'matrix.meta'
-        vocab_file = 'vocab.txt'
-        temp_vocab_file = 'tempvocab.txt'
-        cooccurrence_file = 'cooccurrence'
-        cooccurrence_shuf_file = 'wiki.toy'
-        meta_file = 'meta'
-
         # prepare training data (cooccurrence matrix and vocab)
-        model_dir = os.path.join(wr_path, out_name)
-        meta_dir = os.path.join(model_dir, 'meta')
+        model_dir = join(wr_path, out_name)
```

Comment:
Using full namespace `os.path.join` is preferable. There are many `join`s in Python and its various libraries, and the context makes the code immediately easier to read and understand.